back/app/controllers/web_api/v1/app_configurations_controller.rb

```diff
@@ -10,7 +10,6 @@ def show
   def update
     update_configuration!(app_configuration, config_params)
     side_fx_service = SideFxAppConfigurationService.new
-    side_fx_service.before_update(app_configuration, current_user)
```

Comment:
I think we should leave this one in here. The method is currently a no op, I'd be surprised if I fill it with content, only to find out it's not being called.

Reply:
Good call.